docker/entrypoint-unit-tests-devDocker.sh

```diff
@@ -1,5 +1,6 @@
 #!/bin/sh
 # Run available unittests with a simple setup
+umask 0002
```

Comment:
I don't think you need this as you use the release mode, so it's using entrypoint-uwsgi.sh

Reply:
removed..